src/uct/ib/base/ib_iface.c

```diff
@@ -211,7 +211,8 @@ ucs_config_field_t uct_ib_iface_config_table[] = {
   {"PRIORITY_SLS", "auto",
    "Comma separated list of SLs to use for each priority level.\n"
    "The first SL on the list will be used for priority 0, second one for priority 1, etc.\n"
-   "'auto' will set the same value of sl\n",
+   "'auto' will set the entire array to the value of "
```

Comment:
As I user , I have no idea what "array" is being mentioned here
it should be
"The special value to 'auto' specifies the default interface SL."